goss-testing/tests/common/goss-bond-members-have-correct-mtu.yaml

```diff
@@ -38,4 +38,10 @@ command:
                 "{{$check_bond_members_have_correct_mtu}}" "{{$bond_member_mtu}}"
         exit-status: 0
         timeout: 20000
+        # skip this test on vshasta (the HPC string is not found in this file)
+        {{ if readFile "/etc/os-release" | regexMatch ".*HPC.*" }}
```

Comment:
I'd strongly urge you to check for the existence of `/etc/google_system` instead of parsing the `os-release` file.

The metal OS should always be HPC, but in the distant future that could change due to unavoidable circumstances (it shouldn't change, but if for some reason our HPCM integration causes it to change then this test would fail).

Reply:
I was going to do that, but this:

> NOTE: Goss will error out during during the parsing phase if the file does not exist, no tests will be executed

https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#templates

Perhaps there is a better way to do logic for tests outside of the goss files themselves, but we have several layers of wrapper scripts so I chose this in order to have it work for now.

Reply:
Weird, no way to leverage this in a conditional?:

```
file:
  /etc/google_system:
    exists: true
```

Reply:
Can't you also do straight bash in `command:` ... or does that cause other problems / confuse goss?  (`! test -f /etc/google_system && "{{$logrun}}" -l ...`)

Reply:
I will investigate and report back

Reply:
I think it confuses it, but I will read up some more and report back.

Reply:
This works, but goss doesn't allow combining `--vars` flags.  

```
redbull-ncn-m001-pit:~ # goss -g test.yml --vars <([ -f /etc/google_system ] && echo "vshasta: true" || echo "vshasta: false") validate
Error: found 0 tests, source: test.yml
vshasta $ goss -g test.yml --vars <([ -f /etc/google_system ] && echo "vshasta: true" || echo "vshasta: false") validate
Total Duration: 0.001s
Count: 1, Failed: 0, Skipped: 0
```

I could attempt to expand the above to also read in the other vars files, but that seems even hackier than what I have in this PR